src/System.Net.Http/src/System/Net/Http/Managed/HttpConnection.cs

```diff
@@ -27,7 +27,9 @@ internal sealed partial class HttpConnection : IDisposable
         private const int Expect100TimeoutMilliseconds = 1000;
 
         private static readonly byte[] s_contentLength0NewlineAsciiBytes = Encoding.ASCII.GetBytes("Content-Length: 0\r\n");
+        private static readonly byte[] s_spaceHttp10NewlineAsciiBytes = Encoding.ASCII.GetBytes(" HTTP/1.0\r\n");
         private static readonly byte[] s_spaceHttp11NewlineAsciiBytes = Encoding.ASCII.GetBytes(" HTTP/1.1\r\n");
+        private static readonly byte[] s_spaceHttp20NewlineAsciiBytes = Encoding.ASCII.GetBytes(" HTTP/2.0\r\n");
```

Comment:
This doesn't seem to be used

Reply:
this was added in anticipation of 2.0 support. I can remove it if that is preferred.